.sync.yml

```
@@ -26,6 +26,11 @@ appveyor.yml:
 Rakefile:
   changelog_user: "puppetlabs"
 
+spec/default_facts.yml:
+  extra_facts:
+    pe_server_version: '2016.4.5'
+    puppet_metrics_collector:
+      have_systemd: true
 .rubocop.yml:
   default_configs:
     "Rspec/NamedSubject":
```

```suggestion
    "RSpec/NamedSubject":
```
The casing of this namespace is whats killing your testing

---

What do you mean?  This is currently the only failure that I'll fix soon.

```
spec/defines/pe_metric_spec.rb:33:5: C: [Correctable] RSpec/EmptyLineAfterFinalLet: Add an empty line after the last let.
```